### PR TITLE
Add disable_metrics option

### PR DESCRIPTION
**Pull request by mikker**
Closes elastic/apm-agent-ruby#465

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -318,6 +318,17 @@ NOTE: `global_labels` are supported as of APM server version 7.2. `default_tags`
...
+[[config-disable_metrics]]
+==== `disable_metrics`
+|============
+| Environment                   | `Config` key      | Default | Example
+| `ELASTIC_APM_DISABLE_METRICS` | `disable_metrics` | []      | `"*.cpu.*,system.memory.total"`
+|============
+
+A comma-separated list of dotted metrics names that should not be sent to the APM Server.
+You can use `*` to match multiple metrics.
+
```
Comment (mikker) [inline docs/configuration.asciidoc]: Tiny docs addition, mostly copied from Beni's (again, he's my official ghost writer)

Comment (bmorelli25) [reply docs/configuration.asciidoc]: He's a great 👻writer

File: spec/elastic_apm/config/wildcard_pattern_list_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+          it "#{expectation ? 'matches' : "doesn't match"} #{string}" do
+            expect(subject.match?(string)).to be(expectation)
+          end
+        end
+      end
+    end
+
+    RSpec.describe WildcardPatternList do
+      let(:patterns) { 'foor.*,*.bar' }
```
Comment (mikker) [inline spec/elastic_apm/config/wildcard_pattern_list_spec.rb]:
```suggestion
      let(:patterns) { 'foo.*,*.bar' }
```

Comment (mikker) [reply spec/elastic_apm/config/wildcard_pattern_list_spec.rb]: I love GH

**Comment by mikker** [comment]
CI errors are unrelated.

**Pull request merged**